tests/src/test/java/features/frontendfeatures/landingPage.feature

```diff
@@ -1,11 +1,17 @@
-Feature: Access landing page and verify title
-
-  Background:
-    Given browser is initialized
+Feature: User can navigate to the efiling landing page
 
   @frontend
-  Scenario: Verify if the landing page is available
-    Given user is navigated to the landing page
-    Then page title is verified
-    And browser is closed
-
+  Scenario: Verify valid account guid redirects efiling form page
+    Given user is on the landing page
+    When  user enters invalid account guid "5EDDA22161C043AB9D508158BF4A7828"
+    Then efiling page is displayed and cancel button exists
+  @frontend
+  Scenario: Verify non existing account guid redirects to the efiling form page
+    Given user is on the landing page
+    When  user enters invalid account guid "123"
+    Then efiling page is displayed and cancel button exists
+  @frontend
+  Scenario: Verify existing account guid with conflicts returns error message
+    Given user is on the landing page
+    When  user enters invalid account guid "01E242F88A66475586C2F2B13C5445FA"
```

Comment:
samesies

Reply:
Addressed on the previous comment. Thank you.